cmd/admission-webhook/main.go

```diff
@@ -38,6 +38,7 @@ var (
 )
 
 func init() {
+	flag.CommandLine.Init(os.Args[0], flag.ContinueOnError)
```

Comment:
I think we should add the [missed arguments](https://github.com/pingcap/tidb-operator/blob/master/charts/tidb-operator/templates/admission/admission-webhook-deployment.yaml#L38-L41) into the variables of the `flag`

Reply:
These arguments are used by [openshift/generic-admission-server](https://github.com/pingcap/tidb-operator/blob/master/cmd/admission-webhook/main.go#L76). And it also use `flag.ContinueOnError` in `flagset`to continue to parse arguments when error occur.